// @opentelemetry/exporter-jaeger/build/src/types.js
"use strict";
/*
 * Copyright The OpenTelemetry Authors
 * SPDX-License-Identifier: Apache-2.0
 */
Object.defineProperty(exports, "__esModule", { value: true });
exports.ThriftReferenceType = exports.HTTPSender = exports.ThriftUtils = exports.Utils = exports.UDPSender = void 0;
// Below require is needed as jaeger-client types does not expose the thrift,
// udp_sender, util etc. modules.
/* eslint-disable @typescript-eslint/no-require-imports */
exports.UDPSender = require('jaeger-client/dist/src/reporters/udp_sender').default;
exports.Utils = require('jaeger-client/dist/src/util').default;
exports.ThriftUtils = require('jaeger-client/dist/src/thrift').default;
exports.HTTPSender = require('jaeger-client/dist/src/reporters/http_sender').default;
var ThriftReferenceType;
(function (ThriftReferenceType) {
    ThriftReferenceType["CHILD_OF"] = "CHILD_OF";
    ThriftReferenceType["FOLLOWS_FROM"] = "FOLLOWS_FROM";
})(ThriftReferenceType = exports.ThriftReferenceType || (exports.ThriftReferenceType = {}));
